lib/stages/BaseWriter.hpp

```diff
@@ -98,7 +98,7 @@
 class BaseWriter : public kotekan::Stage {
 public:
     BaseWriter(kotekan::Config& config, const std::string& unique_name,
-               kotekan::bufferContainer& buffer_container);
```

Comment:
Why have `acq_fmt` be assigned directly in the base classes, but have `file_fmt` be passed to the constructor? It would probably be more obvious if they both behaved the same.

Reply:
I've made them both the same now, @jrs65